rust/hook-api/src/config.rs

```diff
@@ -2,7 +2,7 @@ use envconfig::Envconfig;
 
 #[derive(Envconfig)]
 pub struct Config {
-    #[envconfig(from = "BIND_HOST", default = "0.0.0.0")]
+    #[envconfig(from = "BIND_HOST", default = "::")]
```

Comment:
You're switching the socket to IPv6, is that intentional? I could not find a way to tell it to listen to both IPv6 and IPv4 by default.

Reply:
Yeah, by doing `::` it binds to both IPv6 and IPv4 (0.0.0.0 only does IPv4)